resources/assets/js/components/GroupStatsFacts.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div>
+    <h2 class="mt-2 mb-2">{{ translatedFacts }}</h2>
+    <div class="items-container">
+      <StatsValue :count="stats.parties" icon="coffee" size="md" variant="primary" title="groups.events" class="group-stat-events" />
+      <div />
+      <StatsValue :count="stats.pax" icon="participants" size="md" title="groups.participants" class="group-stat-participants" />
+      <div />
+      <StatsValue :count="stats.hours" icon="clock" size="md" title="groups.hours_volunteered" class="group-stat-hours-volunteered" />
+    </div>
+  </div>
+</template>
+<script>
+import StatsValue from './StatsValue'
+
+export default {
+  components: {StatsValue},
+  props: {
+    stats: {
+      required: true,
+      type: Object
+    }
+  },
+  computed: {
+    translatedFacts() {
+      return this.$lang.get('groups.group_facts')
```

Comment:
To avoid "1 events", you need to pluralise this - see for example GroupVolunteers.  Please think about whether this applies to other stats in this change too.

Reply:
Ah thanks yes, good point - this would apply to 'participants' and 'hours volunteered' stats too.

Reply:
I've changed the translatedTitle computed property to make use of the `choice` method from the Laravel JS Localization package, using the roundedCount property to determine whether it's singular or plural.  Given that count is required this seems OK for now.  Works here and looks fine on the existing stats on the events page too - does it look OK to you?

 https://github.com/rmariuzzo/Laravel-JS-Localization#support-for-singular-and-plural-message-based-on-a-count

Reply:
That sounds better than what I did in the pluralize.js mixin that I added.  Should we switch to your model throughout?

Reply:
I think we should - from reading the link above, Lang.js is based on Laravel's Translator class, so all of the existing pluralisation / choice translation templates we have should work as is with Lang.choice.

Reply:
Ok, I've pushed a commit for that to this branch.